desc/protoparse/test-source-info.txt

```diff
@@ -5261,6 +5261,8 @@ desc_test_complex.proto:268:48
  > message_type[9]:
 desc_test_complex.proto:271:1
 desc_test_complex.proto:296:2
+    Trailing comments:
```

Comment:
Without the fix up in `lexer.go`, this trailing comment does not appear -- the comment is incorrectly "lost".